src/GNDStk/JSON/src/write.hpp

```diff
@@ -9,7 +9,7 @@
 
 std::ostream &write(std::ostream &os, const bool decl = true) const
 {
-   (void)decl; // ignored
+   (void)decl; // unused, at least for now
```

Comment:
No worries. I actually use unnamed parameters all the time, when I need the function signature to be a particular thing, but *know* that I'm never going to use the parameter. When I do suspect that I'll use the parameter at some point, I'll generally do the `(void)` thing, precisely because I can easily `grep` for `(void)` and identify such things, when I've dealt with other priorities and am ready to deal with them. (See my comment about the `#if 0` business and `grep`; the same principle is at work here.)